sdk/formrecognizer/azure-ai-formrecognizer/src/main/java/com/azure/ai/formrecognizer/package-info.java

```diff
@@ -2,7 +2,243 @@
+ *     if &#40;merchantPhoneNumberField != null&#41; &#123;
+ *         if &#40;FieldValueType.PHONE_NUMBER == merchantPhoneNumberField.getValue&#40;&#41;.getValueType&#40;&#41;&#41; &#123;
+ *             String merchantAddress = merchantPhoneNumberField.getValue&#40;&#41;.asPhoneNumber&#40;&#41;;
+ *             System.out.printf&#40;&quot;Merchant Phone number: %s, confidence: %.2f%n&quot;,
+ *                 merchantAddress, merchantPhoneNumberField.getConfidence&#40;&#41;&#41;;
+ *         &#125;
+ *     &#125;
+ *
+ *     FormField transactionDateField = recognizedFields.get&#40;&quot;TransactionDate&quot;&#41;;
+ *     if &#40;transactionDateField != null&#41; &#123;
+ *         if &#40;FieldValueType.DATE == transactionDateField.getValue&#40;&#41;.getValueType&#40;&#41;&#41; &#123;
+ *             LocalDate transactionDate = transactionDateField.getValue&#40;&#41;.asDate&#40;&#41;;
+ *             System.out.printf&#40;&quot;Transaction Date: %s, confidence: %.2f%n&quot;,
+ *                 transactionDate, transactionDateField.getConfidence&#40;&#41;&#41;;
+ *         &#125;
+ *     &#125;
+ *
+ *     FormField receiptItemsField = recognizedFields.get&#40;&quot;Items&quot;&#41;;
+ *     if &#40;receiptItemsField != null&#41; &#123;
+ *         System.out.printf&#40;&quot;Receipt Items: %n&quot;&#41;;
+ *         if &#40;FieldValueType.LIST == receiptItemsField.getValue&#40;&#41;.getValueType&#40;&#41;&#41; &#123;
+ *             List&lt;FormField&gt; receiptItems = receiptItemsField.getValue&#40;&#41;.asList&#40;&#41;;
+ *             receiptItems.stream&#40;&#41;
+ *                 .filter&#40;receiptItem -&gt; FieldValueType.MAP == receiptItem.getValue&#40;&#41;.getValueType&#40;&#41;&#41;
+ *                 .map&#40;formField -&gt; formField.getValue&#40;&#41;.asMap&#40;&#41;&#41;
+ *                 .forEach&#40;formFieldMap -&gt; formFieldMap.forEach&#40;&#40;key, formField&#41; -&gt; &#123;
+ *                     if &#40;&quot;Quantity&quot;.equals&#40;key&#41;&#41; &#123;
+ *                         if &#40;FieldValueType.FLOAT == formField.getValue&#40;&#41;.getValueType&#40;&#41;&#41; &#123;
+ *                             Float quantity = formField.getValue&#40;&#41;.asFloat&#40;&#41;;
+ *                             System.out.printf&#40;&quot;Quantity: %f, confidence: %.2f%n&quot;,
+ *                                 quantity, formField.getConfidence&#40;&#41;&#41;;
+ *                         &#125;
+ *                     &#125;
+ *                 &#125;&#41;&#41;;
+ *         &#125;
+ *     &#125;
+ * &#125;
+ * </pre>
+ * <!-- end readme-sample-recognize-receipt-url -->
+ *
+ * <p>
+ * You can also extract data from a local receipt with prebuilt models using the
+ * {@link com.azure.ai.formrecognizer.FormRecognizerClient#beginRecognizeReceipts(java.io.InputStream, long, com.azure.ai.formrecognizer.models.RecognizeReceiptsOptions, com.azure.core.util.Context)}
```

Comment:
For rendering we can have a concise name for this by adding this 
 {@link com.azure.ai.formrecognizer.FormRecognizerClient#beginRecognizeReceipts(java.io.InputStream, long, com.azure.ai.formrecognizer.models.RecognizeReceiptsOptions, com.azure.core.util.Context) beginRecognizeReceipts} method